CHANGES.md

```diff
@@ -117,7 +117,7 @@
   `MutableDocumentRevision` classes have been removed, in order to
   simplify use of API methods. All code using this library will need
   to be updated to use the `DocumentRevision` class. See the updated
-  [CRUD guide](https://github.com/cloudant/sync-android/blob/master/doc/crud.md)
+  [CRUD guide](https://github.com/cloudant/sync-android/blob/master/doc/CrudSamples.java)
```

Comment:
What about changing this link to point to the 1.0.0 tag? That is better than being broken, but not as strange as pointing a 1.0.0 change at a 2.0.0 document.
https://github.com/cloudant/sync-android/blob/1.0.0/doc/crud.md